Project.toml

```diff
@@ -0,0 +1,25 @@
+name = "Blast"
+uuid = "aee4e916-1e40-4aff-8d35-b84b50e5d5e3"
+authors = ["Sofia Chiarenza <sofia.chiarenza@studenti.unimi.it>"]
+version = "0.1.0"
+
+[deps]
+ArtifactUtils = "8b73e784-e7d8-4ea5-973d-377fed4e3bce"
+Artifacts = "56f22d72-fd6d-98f1-02f0-08ddc0907c33"
+BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
```

Comment:
Do we actually need `BenchmarkTools`? I don't think so.
For now we can keep it, but remember to remove it when developing new stuff.